docs/source/install-tutorial.rst

```diff
@@ -85,7 +115,7 @@ A nginx configuration i.e. in ``/etc/nginx/conf.d/bepasty.conf``:
   server {
     listen 80;
     #listen [::]:80; #uncomment this if your server supports IPv6
-    server_name paste.example.org;  # <-- add your domainname here
+    server_name bepasty.example.org;  # <-- add your domainname here
```

Comment:
thanks for wanting to mention bepasty, but paste or pastebin is the more well-known name for that kind of service.

Reply:
I think this needs to be consistent throughout the tutorial.  In the tutorial section that modifies /home/bepasty/bepasty.conf
SITENAME = 'bepasty.example.org'

Therefore, in my humble opinion - whatever is used pasty.example.org or bepasty.example.org has to be consistent so that those that are new to bepasty are not confused.